package.lisp

```diff
@@ -18,7 +18,9 @@
      :assert-false :assert-error :assert-no-error
      :assert-warning :assert-no-warning
      :assert-signal :assert-no-signal
-     :assert-typep
+     :assert-typep :assert-string= :assert-string-equal :assert-string/=
```

Comment:
You probably want to insert a new line after `:assert-typep`.

Reply:
Why not :)

See 2eef316.